projects/plugins/boost/changelog/add-boost-minification

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Getting a bit sick of this setting off the changelogger validity checker. So I'm kicking off a conversation about this. pdWQjU-eR-p2

Reply:
After a discussion with the Crew team, I think we should follow suit and update versions as recommended by the changelogger tool.

pdWQjU-eR-p2

Reply:
I've updated the version in f2dddc5.